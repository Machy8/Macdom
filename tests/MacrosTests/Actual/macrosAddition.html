sectionTitle Some title
quickList Item1 Item2 item3
