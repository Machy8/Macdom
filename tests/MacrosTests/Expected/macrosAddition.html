<h1 class="sectionTitle">
	Some title
</h1>
<ul>
	<li>
		Item1
	</li>
	<li>
		Item2
	</li>
	<li>
		item3
	</li>
</ul>
